client/themes/zenburn.css

```diff
@@ -173,9 +173,9 @@ body {
 }
 
 #form .input {
-	background-color: #434443;
-	border-color: #101010;
-	color: #dcdccc;
+	background-color: #434443 !important;
+	border-color: #101010 !important;
+	color: #dcdccc !important;
```

Comment:
It would be possible, yes. But we're already fighting for the style in the main stylesheet and I didn't feel like making a mess of selectors. But yeah, I should have taken the time but it was late and I wanted to get it fixed ASAP.